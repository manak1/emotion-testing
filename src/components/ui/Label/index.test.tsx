import { composeStories } from "@storybook/testing-react"

import { render } from "~/test/test-utils"

import * as stories from "./index.stories"

const { Default } = composeStories(stories)

describe("Labelのテスト", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Default />)
    expect(asFragment()).toMatchSnapshot()
  })
})
