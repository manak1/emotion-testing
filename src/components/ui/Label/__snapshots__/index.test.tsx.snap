// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Labelのテスト snapshot 1`] = `
<DocumentFragment>
  <label
    class="css-1lu35ut"
  />
</DocumentFragment>
`;
